package.json

```diff
@@ -111,9 +115,8 @@
     "rollup-plugin-node-resolve": "^4.0.0",
     "rollup-plugin-replace": "^2.2.0",
     "rollup-plugin-terser": "^4.0.4",
-    "testcafe": "^1.3.3",
+    "testcafe": "^1.4.3",
```

Comment:
New version of testcafe supports script injection and getting console messages using getBrowserConsoleMessages. Testcafe live comes bundles with the new version.